playbook.yml

```diff
@@ -19,11 +19,13 @@
 - name: Install Redis
   hosts: redis_hosts
   roles:
+    - rdo
     - redis
 
 - name: Install RabbitMQ
   hosts: rabbit_hosts
   roles:
+    - rdo
```

Comment:
Should this be a dependency of the rabbitmq role instead?  Or do you think it should be explicit?  (same question for redis, of course)